packages/frontend/src/features/metricsCatalog/components/MetricsCatalogColumnName.tsx

```diff
@@ -20,7 +20,7 @@ import { type MRT_Row, type MRT_TableInstance } from 'mantine-react-table';
 import { forwardRef, useCallback, useEffect, useState } from 'react';
 import MantineIcon from '../../../components/common/MantineIcon';
 import { useTracking } from '../../../providers/TrackingProvider';
-import MetricIconPlaceholder from '../../../svgs/metrics-catalog-metric-icon.svg?react';
+import { MetricIconPlaceholder } from '../../../svgs/metricsCatalog';
```

Comment:
this should be renamed to `metrics-catalog-metric-icon.svg`

Reply:
This is importing from an `index.js` file where all metrics catalog icons are being barrel exported 💪 
File is actually already named `metric-icon-placeholder`